src/SFA.DAS.EmployerAccounts/Services/CacheStorageService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Threading.Tasks;
+using SFA.DAS.Caches;
+using Newtonsoft.Json;
+using SFA.DAS.EmployerAccounts.Interfaces;
+
+namespace SFA.DAS.EmployerAccounts.Services
+{
+    public class CacheStorageService : ICacheStorageService
+    {
+        private readonly IDistributedCache _distributedCache;
+
+        public CacheStorageService(IDistributedCache distributedCache)
+        {
+            _distributedCache = distributedCache;
+        }
+
+        public async Task SaveToCache<T>(string key, T item, int expirationInHours)
+        {
+            var json = JsonConvert.SerializeObject(item);
+            await _distributedCache.SetCustomValueAsync(key, json, new TimeSpan(1, 0, 0));
```

Comment:
The cache period timespan should come from configuration.

Reply:
The story has mentioned that it is going to always be 1 hr.